mvc/views/razor.rst

```diff
@@ -1,8 +1,283 @@
-.. include:: /../common/stub-topic.txt
+Razor Syntax In MVC 6
+=====================
+By `John Rowley`_
 
-|stub-icon| Razor Syntax
-========================
+---------------
+What is Razor?
+---------------
+Razor is the ASP.NET view engine used in MVC. It allows c# and
+html to be mixed together in `Razor Markup` providing a simple and elegant
+way to render html.
 
-.. include:: /../common/stub-notice.txt
+Razor is the only view engine that MVC ships, but it is possible to use a
+different view engine.
 
-.. _issue: https://github.com/aspnet/Docs/issues/130
+Implicit Expressions
+^^^^^^^^^^^^^^^^^^^^^
+Implicit Expressions provide a simple mechanism to render data. They are simply
+an ``@`` followed by a variable or method. Razor renders the value to at
+runtime. The following markup renders the variable `` GuestName`` inside an H2
+heading element.
+
+.. code-block:: c#
+
+  <h2>@GuestName</h2>
```

Comment:
NTaylorMullen:
Should probably show how the GuestName variable is set to showcase what it’s really doing here. I’d imagine a user reading this document would be completely unfamiliar with Razor and wouldn’t understand how `@GuestName` translated into “Guest-Name”.